docs/product-installation-guide/src/main/asciidoc/roles-users-con.adoc

```diff
@@ -1,23 +1,29 @@
 ifdef::PAM[]
-* `developer`: A developer has access to almost all features and can manage rules, models, process flows, forms, and dashboards. They can manage the asset repository, they can create, build and deploy projects and they can even use Red&nbsp;Hat JBoss&nbsp;Developer&nbsp;Studio to view processes. Only certain administrative functions like creating and cloning a new repository are hidden for the developer role.
+* `developer`: Users with the `developer` role have access to almost all features and can manage rules, models, process flows, forms, and dashboards. They can manage the asset repository, they can create, build, and deploy projects, and they can use Red Hat Developer Studio to view processes. Only certain administrative functions such as creating and cloning a new repository are hidden from users with the `developer` role.
 endif::PAM[]
-* `analyst`: A user with the `analyst` role has access to all high-level features to model
+* `analyst`: Users with the `analyst` role have access to all high-level features. They can model
 ifdef::PAM[]
 and execute their
 endif::PAM[]
-projects. However, *Authoring* -> *Administration* access is unavailable to users with the `analyst` role. Certain lower-level features targeted towards developers, like the *Deployment* -> *Artifact Repository* view are not accessible for this role. However, the *Build & Deploy* button is available for the analyst role while using the Project Editor.
+projects. However, these users cannot add contributors to spaces or delete spaces in the *Design -> Projects* view. Access to the *Deploy -> Execution Servers* view, which is intended for administrators, is not available to users with the analyst role. However, the *Deploy* button is available to these users when they access the Library perspective.
 ifdef::PAM[]
-* `user`: User or a business user work on the business task lists that are used to operate a certain process. A user with this role can access the dashboard and manage processes.
-* `manager`: A manager is a viewer of the system and is interested in statistics around the business processes and their performance, business indicators, and other reporting of the system. A user with this role has access to the BAM only.
+* `user`: Users with the `user` role can work on the Task Inbox list, which contains business tasks that are part of currently running processes. Users with this role can view process and task reports, and manage processes.
+* `manager`: Users with the `manager` role can view reports. These users are usually interested in statistics about the business processes and their performance, business indicators, and other business process related reporting. A user with this role has access only to process and task reports.
```

Comment:
Since it's technically "other business-process-related reporting" (clunky), maybe just "business-related reporting" or "process-related reporting". Re-wording to "other reporting related to business processes" would be good, but then that's kind of redundant with what you've already said.